src/main/java/randoop/test/RegressionCaptureGenerator.java

```diff
@@ -253,4 +277,44 @@ public RegressionChecks generateTestChecks(ExecutableSequence sequence) {
     }
     return checks;
   }
+
+  /**
+   * Returns true if the method:
+   *
+   * <ul>
+   *   <li>has one formal parameter
+   *   <li>has a return type of primitive, boxed primitive, String, or enum
+   *   <li>is side-effect-free
+   * </ul>
+   *
+   * @param tco side-effect-free method as a TypedClassOperation
```

Comment:
Comments should not repeat the type, which is already apparent.  So, cut "as a TypedClassOperation".